src/fu-bluez-device.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (C) 2021 Ricardo Cañuelo <ricardo.canuelo@collabora.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#pragma once
+
+#include <gio/gio.h>
+
+#include "fu-ble-device.h"
+
+#define FU_TYPE_BLUEZ_DEVICE (fu_bluez_device_get_type ())
+G_DECLARE_FINAL_TYPE (FuBluezDevice, fu_bluez_device, FU, BLUEZ_DEVICE, FuBleDevice)
```

Comment:
This should be `G_DECLARE_DERIVABLE_TYPE` no?

Reply:
Yes, and it's worse than that. @hardboprobot pointed out we can't have the FuBleDevice and FuBluezDevice split as otherwise the inorporate design doesn't work; you can't call methods on FuBluezDevice if you only subclass FuBleDevice. I've merged them back together as Google told us on Thursday that they'd be providing a bluez-compatible DBUS API for Chrome.